UI Test B40005 could be automated

- [ ] Missing `Preserve` attribute
- [ ] Test needs to be automated
- [ ] Test crashes on UWP (see #2292)
- [ ] Test crashes on Android, even with `Preserve` attribute (`java.lang.IllegalArgumentException: No view found for id 0xe (unknown) for fragment FragmentContainer{97a4fd6 #5 id=0xe}`) (see #2338)

@PureWeen I installed NUnit 3 Test Adapter VS extension into the VS 2019 (16.0.0) and I was able to run Android tests (I haven't tried iOS yet). However, after a success of running Android UI tests I decided to to re-download Xamarin.Forms sources (fork the repo) and after I've done that I couldn't build the android part of the project :/

**Android**
Firstly, when trying to build `Xamarin.Forms.Core` project it fails with this error:

```
NU1201: Project Xamarin.Forms.ControlGallery.iOS is not compatible with net47 (.NETFramework,Version=v4.7). Project Xamarin.Forms.ControlGallery.iOS supports: xamarinios10 (Xamarin.iOS,Version=v1.0)
NU1201: Project Xamarin.Forms.ControlGallery.iOS is not compatible with net47 (.NETFramework,Version=v4.7) / win. Project Xamarin.Forms.ControlGallery.iOS supports: xamarinios10 (Xamarin.iOS,Version=v1.0)
NU1201: Project Xamarin.Forms.ControlGallery.iOS is not compatible with net47 (.NETFramework,Version=v4.7) / win-x64. Project Xamarin.Forms.ControlGallery.iOS supports: xamarinios10 (Xamarin.iOS,Version=v1.0)
NU1201: Project Xamarin.Forms.ControlGallery.iOS is not compatible with net47 (.NETFramework,Version=v4.7) / win-x86. Project Xamarin.Forms.ControlGallery.iOS supports: xamarinios10 (Xamarin.iOS,Version=v1.0)
NuGet package restore failed. Please see Error List window for detailed warnings and errors.
1>------ Build started: Project: Xamarin.Forms.Platform, Configuration: Debug Any CPU ------
1>Xamarin.Forms.Platform -> C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform\bin\Debug\netstandard1.0\Xamarin.Forms.Platform.dll
1>Xamarin.Forms.Platform -> C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform\bin\Debug\netstandard2.0\Xamarin.Forms.Platform.dll
2>------ Build started: Project: Xamarin.Forms.Core, Configuration: Debug Any CPU ------
2>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Core\Xamarin.Forms.Core.csproj(31,6): error MSB4062: The "XFCorePostProcessor.Tasks.FixXFCoreAssembly" task could not be loaded from the assembly C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Core\..\XFCorePostProcessor.Tasks\bin\Debug\net461\XFCorePostProcessor.Tasks.dll. Could not load file or assembly 'file:///C:\Programming\SourceCode\Xamarin.Forms\XFCorePostProcessor.Tasks\bin\Debug\net461\XFCorePostProcessor.Tasks.dll' or one of its dependencies. The system cannot find the file specified. Confirm that the <UsingTask> declaration is correct, that the assembly and all its dependencies are available, and that the task contains a public class that implements Microsoft.Build.Framework.ITask.
2>Done building project "Xamarin.Forms.Core.csproj" -- FAILED.
2>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Core\Xamarin.Forms.Core.csproj(31,6): error MSB4062: The "XFCorePostProcessor.Tasks.FixXFCoreAssembly" task could not be loaded from the assembly C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Core\..\XFCorePostProcessor.Tasks\bin\Debug\net461\XFCorePostProcessor.Tasks.dll. Could not load file or assembly 'file:///C:\Programming\SourceCode\Xamarin.Forms\XFCorePostProcessor.Tasks\bin\Debug\net461\XFCorePostProcessor.Tasks.dll' or one of its dependencies. The system cannot find the file specified. Confirm that the <UsingTask> declaration is correct, that the assembly and all its dependencies are available, and that the task contains a public class that implements Microsoft.Build.Framework.ITask.
2>Done building project "Xamarin.Forms.Core.csproj" -- FAILED.
========== Build: 1 succeeded, 1 failed, 0 up-to-date, 0 skipped ==========
```

The bizarre thing is that if I try building it for the second time the build succeeds. Even though I was able to pass this point I got stuck on trying to build `Xamarin.Forms.Platform.Android` project. I thought maybe it's related to some latest changes that are breaking the build. I then grabbed slightly older commit (e.g. `32da34df0e871fec8ace5cb1fd72ed2d2b7899d6`), that has a green check next to it in the commit history, indicating that the build passes. Unfortunately no luck there. Still getting errors like so:

```
1>------ Build started: Project: Xamarin.Forms.Platform.Android, Configuration: Debug Any CPU ------
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\VisualElementRenderer.cs(14,58,14,72): error CS0246: The type or namespace name 'FormsViewGroup' could not be found (are you missing a using directive or an assembly reference?)
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\AppCompat\ButtonRenderer.cs(185,8,185,24): error CS0246: The type or namespace name 'IOnClickListener' could not be found (are you missing a using directive or an assembly reference?)
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\AppCompat\ButtonRenderer.cs(187,8,187,24): error CS0246: The type or namespace name 'IOnTouchListener' could not be found (are you missing a using directive or an assembly reference?)
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\AppCompat\ButtonRenderer.cs(185,8,185,24): error CS0538: 'IOnClickListener' in explicit interface declaration is not an interface
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\AppCompat\ButtonRenderer.cs(187,8,187,24): error CS0538: 'IOnTouchListener' in explicit interface declaration is not an interface
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\AppCompat\ButtonRenderer.cs(16,39,16,61): error CS0535: 'ButtonRenderer' does not implement interface member 'View.IOnClickListener.OnClick(View)'
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\AppCompat\ButtonRenderer.cs(16,63,16,85): error CS0535: 'ButtonRenderer' does not implement interface member 'View.IOnTouchListener.OnTouch(View, MotionEvent)'
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\AppCompat\NavigationPageRenderer.cs(270,27,270,45): error CS0115: 'NavigationPageRenderer.OnAttachedToWindow()': no suitable method found to override
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\AppCompat\NavigationPageRenderer.cs(291,27,291,47): error CS0115: 'NavigationPageRenderer.OnDetachedFromWindow()': no suitable method found to override
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\AppCompat\NavigationPageRenderer.cs(33,97,33,113): error CS0535: 'NavigationPageRenderer' does not implement interface member 'IJavaObject.Handle'
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\NativeViewWrapperRenderer.cs(66,27,66,36): error CS0115: 'NativeViewWrapperRenderer.OnMeasure(int, int)': no suitable method found to override
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\AppCompat\TabbedPageRenderer.cs(214,27,214,45): error CS0115: 'TabbedPageRenderer.OnAttachedToWindow()': no suitable method found to override
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\AppCompat\TabbedPageRenderer.cs(222,27,222,47): error CS0115: 'TabbedPageRenderer.OnDetachedFromWindow()': no suitable method found to override
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\AppCompat\TabbedPageRenderer.cs(24,71,24,103): error CS0535: 'TabbedPageRenderer' does not implement interface member 'IJavaObject.Handle'
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\AppCompat\CarouselPageRenderer.cs(82,27,82,45): error CS0115: 'CarouselPageRenderer.OnAttachedToWindow()': no suitable method found to override
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\AppCompat\CarouselPageRenderer.cs(91,27,91,47): error CS0115: 'CarouselPageRenderer.OnDetachedFromWindow()': no suitable method found to override
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\AppCompat\CarouselPageRenderer.cs(10,75,10,106): error CS0535: 'CarouselPageRenderer' does not implement interface member 'IJavaObject.Handle'
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\Renderers\ButtonRenderer.cs(301,39,301,55): error CS0246: The type or namespace name 'IOnClickListener' could not be found (are you missing a using directive or an assembly reference?)
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\Renderers\ButtonRenderer.cs(313,39,313,55): error CS0246: The type or namespace name 'IOnTouchListener' could not be found (are you missing a using directive or an assembly reference?)
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\Renderers\CarouselPageRenderer.cs(42,27,42,45): error CS0115: 'CarouselPageRenderer.OnAttachedToWindow()': no suitable method found to override
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\Renderers\CarouselPageRenderer.cs(57,27,57,47): error CS0115: 'CarouselPageRenderer.OnDetachedFromWindow()': no suitable method found to override
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\Renderers\CarouselPageRenderer.cs(110,27,110,36): error CS0115: 'CarouselPageRenderer.OnMeasure(int, int)': no suitable method found to override
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\ViewRenderer.cs(31,114,31,142): error CS0535: 'ViewRenderer<TView, TNativeView>' does not implement interface member 'IJavaObject.Handle'
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\Platform.cs(1277,44,1277,60): error CS0246: The type or namespace name 'IOnTouchListener' could not be found (are you missing a using directive or an assembly reference?)
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\Platform.cs(1197,4,1197,20): error CS0246: The type or namespace name 'IOnTouchListener' could not be found (are you missing a using directive or an assembly reference?)
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\Renderers\PageRenderer.cs(40,27,40,45): error CS0115: 'PageRenderer.OnAttachedToWindow()': no suitable method found to override
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\Renderers\PageRenderer.cs(49,27,49,47): error CS0115: 'PageRenderer.OnDetachedFromWindow()': no suitable method found to override
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\Renderers\TabbedRenderer.cs(42,27,42,45): error CS0115: 'TabbedRenderer.OnAttachedToWindow()': no suitable method found to override
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\Renderers\TabbedRenderer.cs(48,27,48,47): error CS0115: 'TabbedRenderer.OnDetachedFromWindow()': no suitable method found to override
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\VisualElementRenderer.cs(36,24,36,36): error CS0115: 'VisualElementRenderer<TElement>.OnTouchEvent(MotionEvent)': no suitable method found to override
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\VisualElementRenderer.cs(41,24,41,45): error CS0115: 'VisualElementRenderer<TElement>.OnInterceptTouchEvent(MotionEvent)': no suitable method found to override
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\VisualElementRenderer.cs(53,24,53,42): error CS0115: 'VisualElementRenderer<TElement>.DispatchTouchEvent(MotionEvent)': no suitable method found to override
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\VisualElementRenderer.cs(156,25,156,36): error CS0115: 'VisualElementRenderer<TElement>.FocusSearch(View, FocusSearchDirection)': no suitable method found to override
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\VisualElementRenderer.cs(260,27,260,34): error CS0115: 'VisualElementRenderer<TElement>.Dispose(bool)': no suitable method found to override
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\VisualElementRenderer.cs(357,27,357,35): error CS0115: 'VisualElementRenderer<TElement>.OnLayout(bool, int, int, int, int)': no suitable method found to override
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\VisualElementRenderer.cs(14,74,14,96): error CS0535: 'VisualElementRenderer<TElement>' does not implement interface member 'IDisposable.Dispose()'
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\Renderers\ListViewRenderer.cs(105,27,105,45): error CS0115: 'ListViewRenderer.OnAttachedToWindow()': no suitable method found to override
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\Renderers\ListViewRenderer.cs(114,27,114,47): error CS0115: 'ListViewRenderer.OnDetachedFromWindow()': no suitable method found to override
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\Renderers\NavigationRenderer.cs(97,27,97,45): error CS0115: 'NavigationRenderer.OnAttachedToWindow()': no suitable method found to override
1>C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Platform.Android\Renderers\NavigationRenderer.cs(103,27,103,47): error CS0115: 'NavigationRenderer.OnDetachedFromWindow()': no suitable method found to override
```

I'm not sure what's going on there...

**UWP**

I also tried running it against UWP but it's constantly failing with an error:

```
[14/04/2019 5:10:11 PM Informational] Run started: C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Core.Windows.UITests\bin\Debug\Xamarin.Forms.Core.Windows.UITests.dll
[14/04/2019 5:10:13 PM Error] SetUp failed for test fixture Xamarin.Forms.Controls.Issues.IssuesSetup
[14/04/2019 5:10:13 PM Error] SetUp : NUnit.Framework.InconclusiveException : App did not start for some reason: OpenQA.Selenium.WebDriverException: Unexpected error. System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:4723
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at OpenQA.Selenium.Appium.Service.AppiumCommandExecutor.Execute(Command commandToExecute)
   at OpenQA.Selenium.Remote.RemoteWebDriver.Execute(String driverCommandToExecute, Dictionary`2 parameters)
   at OpenQA.Selenium.Remote.RemoteWebDriver.UnpackAndThrowOnError(Response errorResponse)
   at OpenQA.Selenium.Remote.RemoteWebDriver.Execute(String driverCommandToExecute, Dictionary`2 parameters)
   at OpenQA.Selenium.Appium.AppiumDriver`1.Execute(String driverCommandToExecute, Dictionary`2 parameters)
   at OpenQA.Selenium.Remote.RemoteWebDriver.StartSession(ICapabilities desiredCapabilities)
   at OpenQA.Selenium.Remote.RemoteWebDriver..ctor(ICommandExecutor commandExecutor, ICapabilities desiredCapabilities)
   at OpenQA.Selenium.Appium.AppiumDriver`1..ctor(Uri remoteAddress, ICapabilities appiumOptions)
   at OpenQA.Selenium.Appium.Windows.WindowsDriver`1..ctor(Uri remoteAddress, AppiumOptions AppiumOptions)
   at Xamarin.Forms.Core.UITests.WindowsTestBase.CreateWindowsDriver() in C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Core.Windows.UITests\WindowsTestBase.cs:line 30
   at Xamarin.Forms.Core.UITests.WindowsTestBase.ConfigureApp() in C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Core.Windows.UITests\WindowsTestBase.cs:line 20
   at Xamarin.Forms.Controls.AppSetup.InitializeUWPApp() in C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Controls.Issues\Xamarin.Forms.Controls.Issues.Shared\TestPages\TestPages.cs:line 119
   at Xamarin.Forms.Controls.AppSetup.InitializeApp() in C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Controls.Issues\Xamarin.Forms.Controls.Issues.Shared\TestPages\TestPages.cs:line 51
   at Xamarin.Forms.Controls.AppSetup.Setup(Type pageType) in C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Controls.Issues\Xamarin.Forms.Controls.Issues.Shared\TestPages\TestPages.cs:line 220
[14/04/2019 5:10:13 PM Error]    at NUnit.Framework.Assert.Inconclusive(String message, Object[] args)
   at NUnit.Framework.Assert.Inconclusive(String message)
   at Xamarin.Forms.Controls.AppSetup.Setup(Type pageType) in C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Controls.Issues\Xamarin.Forms.Controls.Issues.Shared\TestPages\TestPages.cs:line 224
   at Xamarin.Forms.Controls.Issues.IssuesSetup.RunBeforeAnyTests() in C:\Programming\SourceCode\Xamarin.Forms\Xamarin.Forms.Controls.Issues\Xamarin.Forms.Controls.Issues.Shared\TestPages\TestPages.cs:line 646
[14/04/2019 5:10:13 PM Informational] NUnit VS Adapter 2.1.1.0 executing tests is finished
[14/04/2019 5:10:13 PM Informational] ========== Run test finished: 1 run (0:00:11.0676725) ==========
```

Please let me know if there's a better way of communicating with you (or this github convo is fine?) or anyone else from the team, as I would really like to get this change in and potentially start helping with other issue/features etc.

--- a/Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla40005.cs
+++ b/Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla40005.cs
@@ -11,7 +11,6 @@ using Xamarin.Forms.Core.UITests;
 
 namespace Xamarin.Forms.Controls.Issues
 {
-
 #if UITEST
 	[NUnit.Framework.Category(UITestCategories.Navigation)]
 	[NUnit.Framework.Category(UITestCategories.ManualReview)]
@@ -21,6 +20,10 @@ namespace Xamarin.Forms.Controls.Issues
 	public class Bugzilla40005 : TestContentPage // or TestMasterDetailPage, etc ...
 	{
 		public const string GoToPage2 = "Go to Page 2";
+		public const string PageOneLabel = "Page 1";
+		public const string PageTwoLabel = "Page 2";
+		public const string InsertedPageLabel = "Inserted page";
+		public const string TestInstructions = "Click " + GoToPage2 + " and you should still see a back bar button";
 
 		public Bugzilla40005()
 		{
@@ -37,7 +40,7 @@ namespace Xamarin.Forms.Controls.Issues
 
 			public Page1()
 			{
-				Button btn = new Button()
+				var btn = new Button()
 				{
 					Text = GoToPage2
 				};
@@ -51,15 +54,17 @@ namespace Xamarin.Forms.Controls.Issues
 					VerticalOptions = LayoutOptions.Center,
 					Children =
 					{
-						new Label {
+						new Label
+						{
 							HorizontalTextAlignment = TextAlignment.Center,
-							Text = "Page 1"
+							Text = PageOneLabel
 						},
 						btn,
-						new Label {
+						new Label
+						{
 							HorizontalTextAlignment = TextAlignment.Center,
-							Text = $"Click {GoToPage2} and you should still see a back bar button"
-						},
+							Text = TestInstructions
+						}
 					}
 				};
 			}
@@ -76,11 +81,10 @@ namespace Xamarin.Forms.Controls.Issues
 
 			protected override bool OnBackButtonPressed()
 			{
-				Debug.WriteLine("Hardware BackButton Pressed on Page1");
+				Debug.WriteLine($"Hardware BackButton Pressed on {PageOneLabel}");
 				return base.OnBackButtonPressed();
 			}
 		}
-
 
 		public class InsertedPage : ContentPage
 		{
@@ -89,18 +93,20 @@ namespace Xamarin.Forms.Controls.Issues
 				Content = new StackLayout
 				{
 					VerticalOptions = LayoutOptions.Center,
-					Children = {
-					new Label {
-						HorizontalTextAlignment = TextAlignment.Center,
-						Text = "Inserted page"
+					Children =
+					{
+						new Label
+						{
+							HorizontalTextAlignment = TextAlignment.Center,
+							Text = InsertedPageLabel
+						}
 					}
-				}
 				};
 			}
 
 			protected override bool OnBackButtonPressed()
 			{
-				Debug.WriteLine("Hardware BackButton Pressed on InsertedPage");
+				Debug.WriteLine($"Hardware BackButton Pressed on {InsertedPageLabel}");
 				return base.OnBackButtonPressed();
 			}
 		}
@@ -112,20 +118,34 @@ namespace Xamarin.Forms.Controls.Issues
 				Content = new StackLayout
 				{
 					VerticalOptions = LayoutOptions.Center,
-					Children = {
-					new Label {
-						HorizontalTextAlignment = TextAlignment.Center,
-						Text = "Page 2"
+					Children =
+					{
+						new Label
+						{
+							HorizontalTextAlignment = TextAlignment.Center,
+							Text = PageTwoLabel
+						}
 					}
-				}
 				};
 			}
 
 			protected override bool OnBackButtonPressed()
 			{
-				Debug.WriteLine("Hardware BackButton Pressed on Page2");
+				Debug.WriteLine($"Hardware BackButton Pressed on {PageTwoLabel}");
 				return base.OnBackButtonPressed();
 			}
 		}
+
+#if UITEST
+		[Test]
+		public void Bugzilla40005Test()
+		{
+			RunningApp.WaitForElement(q => q.Marked(PageOneLabel));
+			RunningApp.Tap(q => q.Marked(GoToPage2));
+			RunningApp.WaitForElement(q => q.Marked(PageTwoLabel));
+			RunningApp.Back();
+			RunningApp.WaitForElement(q => q.Marked(PageOneLabel));
+		}
+#endif
 	}
 }

